projects/ccf-rui/src/app/shared/components/gender-selector/gender-selector.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
```

Comment:
This entire component and side-selector component are duplicates. Please create a single generic component (named `labeled-side-toggle` perhaps)